bitnami/etcd/values.yaml

```diff
@@ -537,6 +537,59 @@ volumePermissions:
     limits: {}
     requests: {}
 
+## Network Policy configuration
+## ref: https://kubernetes.io/docs/concepts/services-networking/network-policies/
+##
```

Comment:
Add a new section to avoid the parameters to be listed under "Volume Permissions parameters" in the README.md:
```suggestion
## @section Network Policy parameters
## ref: https://kubernetes.io/docs/concepts/services-networking/network-policies/
##
```
Then, re-run the readme-generator tool.

Reply:
Fixed